pkg/loki/modules.go

```diff
@@ -336,6 +346,16 @@ func (t *Loki) initQueryFrontend() (err error) {
 	t.server.HTTP.Handle("/api/prom/series", frontendHandler)
 	// fallback route
 	t.server.HTTP.PathPrefix("/").Handler(frontendHandler)
+
+	// Once the execution reaches this point, all initialization has been
+	// done and the query-frontend is ready to serve, so we're just returning a 200.
+	t.server.HTTP.Path("/ready").Handler(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think this is interesting to have this for the frontend too but not sure we want to do it in this PR. Mainly because I think it might be different.

Currently it returns

```
curl localhost:9090/ready
no org id
```
So we might just need to change that. but for now let's remove it so we can close the issue for the table manager.

Reply:
Ok. Will remove frontend

Reply:
Thanks you can create an issue though I think it's a good thing to fix.

Reply:
Sure